org.dupot.littleadventure.yml

```diff
@@ -0,0 +1,29 @@
+id: org.dupot.littleadventure
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: littleAdventure.x86_64
+finish-args:
+  - --share=ipc
+  - --socket=pulseaudio
+  - --socket=x11
+  - --device=dri
+modules:
+  - name: littleAdventure
+    buildsystem: simple
+    build-commands:
+      - install -Dm555 export/linux/littleAdventure.x86_64 /app/bin/littleAdventure.x86_64
+      - install -Dm555 export/linux/littleAdventure.pck /app/bin/littleAdventure.pck
+      - install -Dm644 512x512.png /app/share/icons/hicolor/512x512/apps/org.dupot.littleadventure.png
+      - install -Dm644 org.dupot.littleadventure.desktop /app/share/applications/org.dupot.littleadventure.desktop
+      - install -Dm644 org.dupot.littleadventure.appdata.xml /app/share/metainfo/org.dupot.littleadventure.appdata.xml
+    sources:
+      - type: git
+        url: https://github.com/imikado/dupotLittleAdventure/
+        commit: 884732e35732c103c1d11c3864bb6a2110ee010f
+      - type: file
+        path: 512x512.png
+      - type: file
+        path: org.dupot.littleadventure.desktop
+      - type: file
+        path: org.dupot.littleadventure.appdata.xml
```

Comment:
all these files are also in your repository so you shouldn't need these here.

To clarify: please remove them and install the one in the repository

Reply:
i remove files and update remote one (i had mistake on it)
and i update my commit id

thanks for advice